### PR TITLE
Add South Korea 2022 upcoming conference

### PR DESCRIPTION
**Pull request by M1zz**

### CODE REVIEW

File: _data/conferences.yml [hunk 1]
```diff
@@ -23,6 +23,36 @@
...
+
+- name: Let us: Go!
+  link: https://let-us-go-2022-fall.vercel.app
+  start: 2022-11-05
+  end: 2020-11-05
+  location: 🇰🇷 Seoul, South Korea
+  cocoa-only: true
+
+  https://let-us-go-2022-fall.vercel.app
+
```
Comment (davedelong) [inline _data/conferences.yml:55]: This looks like a typo?

File: _data/conferences.yml [hunk 2]
```diff
@@ -23,6 +23,36 @@
...
 
+- name: adiOS
+  link: tbd
+  start: 2022-12-18
+  end: 2020-12-18
+  location: 🇰🇷 Seoul, South Korea
+  cocoa-only: true
+
+- name: Let’Swift
+  link: http://letswift.kr/2022/
```
Comment (davedelong) [inline _data/conferences.yml:34]: Is there an HTTPS version of the website?

**Review by davedelong — CHANGES_REQUESTED** [review]

File: _data/conferences.yml (hunk 1 above)
Comment (M1zz) [reply _data/conferences.yml]: sorry my documentation tool is not good I deleted typo.

File: _data/conferences.yml (hunk 2 above)
Comment (M1zz) [reply _data/conferences.yml:34]: I searched it and they have! I will update it

**Review by davedelong — APPROVED** [review]
Thank you very much @M1zz! :)

**Comment by cgrail** [comment]
@M1zz  I've also added the conference to our site: https://confs.tech
https://github.com/tech-conferences/conference-data/pull/4808
https://github.com/tech-conferences/conference-data/pull/4809
https://github.com/tech-conferences/conference-data/pull/4810

**Comment by M1zz** [comment]
Thanks! Can I add previous event for archiving?

**Comment by davedelong** [comment]
Sure! Submit a new PR with the historical information and we'll merge it.

**Pull request merged**